react/SliderLayout.tsx

```diff
@@ -13,6 +13,8 @@ const SliderLayout: StorefrontFunctionComponent<
   showNavigationArrows = 'always',
   showPaginationDots = 'always',
   usePagination = true,
+  fullWidth = true,
+  arrowIconSize = 25,
```

Comment:
It might be nice to make this 25 a constant so we can avoid magic numbers

Reply:
Uuuhm... I'm not sure if that would be much useful in this case, since I'm setting the default value for this prop here, and everywhere else `arrowIconSize` is used it already has a value attributed to it, so it's used as `arrowIconSize`.